### PR TITLE
Ampli Doc Header Update

### PR DESCRIPTION
**Pull request by bohan-amplitude**
# Dev Docs PR

## Description

Standardize the 2 Bata SDK docs header:
TS Node
TS RN

Headers:

SDK description
Ampli Example Apps
Itly Deprecated warning
Beta SDK not supported warning ( RN & Node)

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [x] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [x] My documentation follows the style guidelines of this project.
- [x] I previewed my documentation on a local server using `mkdocs serve`.
- [x] Running `mkdocs serve` didn't generate any failures.
- [x] I have performed a self-review of my own documentation.

@amplitude-dev-docs
@caseyamp

### CODE REVIEW

File: docs/data/sdks/android-kotlin/ampli.md [hunk 1]
```diff
@@ -3,16 +3,17 @@ title: Android Ampli Wrapper
...
 
 In Kotlin and Java, the tracking library exposes a type-safe function for every event in your team’s tracking plan. The function’s arguments correspond to the event’s properties and are strongly typed to allow for code completion and compile-time checks.
 
-!!!tip
+!!!example "Ampli Example"
+
+    See example apps that use the Android [Java](https://github.com/amplitude/ampli-examples/tree/main/android/java/v2/AmpliApp) and [Kotlin](https://github.com/amplitude/ampli-examples/tree/main/android/kotlin/v2/AmpliApp) runtimes.
 
-    See example apps that use the Android Java and Kotlin runtimes on [GitHub](https://github.com/amplitude/ampli-examples/tree/main/android).
+!!!danger "Deprecated Itly runtime"
```
Comment (justin-fiedler) [inline docs/data/sdks/android-kotlin/ampli.md:15]: I think @caseyamp is reserving `danger` for breaking changes only. Maybe we should change back to a note. Not to many folks on Itly these days so I don't think we need to call massive attention to it.

File: docs/data/sdks/android/ampli.md [hunk 2]
```diff
@@ -3,16 +3,16 @@ title: Android Ampli Wrapper
...
 
-!!!note
-    This page covers the Android Java and Kotlin runtimes. All (Itly) runtimes are deprecated. If you are still using an (Itly) runtime, see the **[migration guide](#migrating-from-previous-version)** to upgrade to the newest runtime. Docs for the Itly version are available **[here](browser)**.
 
 Amplitude Data supports tracking analytics events from Android apps written in Kotlin and Java.
 
 In Kotlin and Java, the tracking library exposes a type-safe function for every event in your team’s tracking plan. The function’s arguments correspond to the event’s properties and are strongly typed to allow for code completion and compile-time checks.
 
-!!!tip
+!!!example "Ampli Example"
```
Comment (justin-fiedler) [inline docs/data/sdks/android/ampli.md:11]: To keep this similar to the core SDK docs I was thinking we could display this like `SDK Resources`. WDYT? cc @caseyamp 

```
Ampli Resources
* Android Java Ampli Examples
* Android Kotlin Ampli Examples
```

image

**Review by justin-fiedler — CHANGES_REQUESTED** [review]

**Review by caseyamp — CHANGES_REQUESTED** [review]

File: docs/data/sdks/android-kotlin/ampli.md (hunk 1 above)
Comment (bohan-amplitude) [reply docs/data/sdks/android-kotlin/ampli.md]: changed to new deprecated admonition

File: docs/data/sdks/android/index.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ icon: material/android
 
 This is the official documentation for the Amplitude Analytics Android SDK.
 
-!!!attention "Legacy SDK"
+!!!deprecated "Legacy SDK"
     This SDK is legacy and only continue to receive bug fixes until deprecated. Checkout and upgrade to the [Android Kotlin SDK](../android-kotlin) which supports plugins, SDK integrations and more.
```
Comment (caseyamp) [inline docs/data/sdks/android/index.md:13]:
```suggestion
    This is a legacy SDK and will only receive bug fixes until deprecation. Upgrade to the [Android Kotlin SDK](../android-kotlin) which supports plugins, SDK integrations, and more.
```

**Review by caseyamp — COMMENTED** [review]
Only had one suggestion, thanks for all your work on this @bohan-amplitude

File: docs/data/sdks/android/index.md (hunk 3 above)
Comment (bohan-amplitude) [reply docs/data/sdks/android/index.md]: Changed for android and other SDKs that has Legacy block

**Review by caseyamp — APPROVED** [review]

File: docs/data/sdks/android-kotlin/ampli.md [hunk 4]
```diff
@@ -3,16 +3,17 @@ title: Android Ampli Wrapper
...
 
 In Kotlin and Java, the tracking library exposes a type-safe function for every event in your team’s tracking plan. The function’s arguments correspond to the event’s properties and are strongly typed to allow for code completion and compile-time checks.
 
-!!!tip
+!!!info "Ampli Resources"
+    - [Android Java Ampli Examples](https://github.com/amplitude/ampli-examples/tree/main/android/java/v2/AmpliApp)
+    - [Android Kotlin Ampli Examples](https://github.com/amplitude/ampli-examples/tree/main/android/kotlin/v2/AmpliApp)
 
-    See example apps that use the Android Java and Kotlin runtimes on [GitHub](https://github.com/amplitude/ampli-examples/tree/main/android).
+!!!deprecated "Deprecated Itly runtime"
```
Comment (justin-fiedler) [inline docs/data/sdks/android-kotlin/ampli.md:15]: I think this deprecation warning is confusing/aggressive. Most customers will not be coming from an Itly runtime, and this warning makes it look like the SDK on the current page is deprecated vs Itly. Can we make a difference admon for this? Or maybe revert it to a Note? cc @caseyamp 
image

**Review by justin-fiedler — CHANGES_REQUESTED** [review]

File: docs/data/sdks/android-kotlin/ampli.md (hunk 4 above)
Comment (caseyamp) [reply docs/data/sdks/android-kotlin/ampli.md:15]: I honestly have always thought the Itly deprecation notices aren't necessary -- like you said, most aren't coming from an Itly runtime so it's weird to even mention it on a non-itly SDK. If they're using Itly, why are they on this page? We should have these notices only on the deprecated runtime docs, IMO. @justin-fiedler

Comment (bohan-amplitude) [reply docs/data/sdks/android-kotlin/ampli.md:15]: The deprecated runtime docs don't have nav bar entry. I think these notices are the only entries for deprecated runtime docs now. Changing back to !!!note is better for now @justin-fiedler @caseyamp

Comment (caseyamp) [reply docs/data/sdks/android-kotlin/ampli.md:15]: I'm okay with that option @bohan-amplitude.

Comment (justin-fiedler) [reply docs/data/sdks/android-kotlin/ampli.md]: I think a future improvement could be to make a dedicated section for Itly migration and then remove mentions to Itly in the current SDKs. For now I think a note is ok.

**Review by justin-fiedler — APPROVED** [review]

**Comment by justin-fiedler** [comment]
@bohan-amplitude I need these changes for Ampli v2 docs. Merging.

**Pull request merged**